src/compiler/moduleNameResolver.ts

```diff
@@ -1368,7 +1370,8 @@ namespace ts {
             }
             const resolved = forEach(paths[matchedPatternText], subst => {
                 const path = matchedStar ? subst.replace("*", matchedStar) : subst;
-                const candidate = normalizePath(combinePaths(baseDirectory, path));
+                // When baseUrl is not specified, the command line parser resolves relative paths to the config file location.
+                const candidate = normalizePath(combinePaths(baseDirectory || "", path));

```

Comment:
why do you have `||` here. baseDirectory is suppose to be string so you shouldnt need to do this here.

Reply:
You’re right, this is left over from a previous iteration 👍